src/tests/run_manually.rs

```diff
@@ -210,9 +210,13 @@ pub(crate) fn run_and_try_create_witness_for_extended_state(
             ..Default::default()
         },
     )
+    .expect("Run failed")
```

Comment:
It will silence exception message from run, isn't it?

Reply:
My bad

Reply:
it should attach it:
(from documentation):
```
let x: Result<u32, &str> = Err("emergency failure");
x.expect("Testing expect"); // panics with `Testing expect: emergency failure`
```